sw/device/lib/base/memory.c

```diff
@@ -0,0 +1,35 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/memory.h"
+
+extern inline uint32_t read_32(void *);
```

Comment:
These should just be `extern`. I got this wrong in the first version of the C11 style guide and updated it.